assessment/collectors/app_migration_analysis.go

```diff
@@ -138,13 +151,17 @@ func (m *MigrationSummarizer) MigrationCodeConversionInvoke(
 	if err != nil {
 		return "", err
 	}
+	logger.Log.Debug("Token: ",
+		zap.Int32("Prompt Token:", resp.UsageMetadata.PromptTokenCount),
+		zap.Int32("Candidate Token:", resp.UsageMetadata.CandidatesTokenCount),
+		zap.Int32("Total Token:", resp.UsageMetadata.TotalTokenCount))
 
 	var response string
 	if p, ok := resp.Candidates[0].Content.Parts[0].(genai.Text); ok {
 		response = string(p)
 	}
 
-	response = ParseJSONWithRetries(m.modelPro, prompt, response, 2, identifier)
+	response = ParseJSONWithRetries(m.modelFlash, prompt, response, identifier)
```

Comment:
is flash working better than pro here ? That seems odd.

Reply:
It's not better but there were not much difference in the response parsing with Pro.